src/udev/udev-rules.c

```diff
@@ -1168,6 +1168,21 @@ static int parse_line(char **line, char **ret_key, char **ret_attr, UdevRuleOper
         return 1;
 }
 
+static void check_tokens_order(UdevRuleLine *rule_line) {
+        assert(rule_line);
+
+        bool has_result = false;
```

Comment:
Nowadays we dropped such rule, but still we usually first declare variables then assertions, like old C.